examples/grpc_proxying/README.md

```diff
@@ -0,0 +1,97 @@
+# Example - Add an existing gRPC service to the Python SDK gRPC App extension
+
+This example creates a gRPC service using the protobuf file and adds it to the Python SDK gRPC App extension. Using add_external_service, we can add the `HelloWorld` servicer. This way we can use gRPC Proxying feature of Dapr and at the same time have a full access to other features that come with Python SDK, such as pubsub.
+
+> **Note:** Make sure to use the latest proto bindings
+
+## Pre-requisites
+
+- [Dapr CLI and initialized environment](https://docs.dapr.io/getting-started)
+- [Install Python 3.7+](https://www.python.org/downloads/)
+
+## Install Dapr python-SDK
+
+<!-- Our CI/CD pipeline automatically installs the correct version, so we can skip this step in the automation -->
+
+```bash
+pip3 install dapr dapr-ext-grpc
+```
+
+## Running in self-hosted mode
+
+Run the following command in a terminal/command-prompt:
+
+<!-- STEP -->
```

Comment:
So these step annotation comments are not just for show, we actually have some automation which validates our examples with every run. It is capable of running your \`\`\`bash ... \`\`\` codeblocks and checking stdout. This avoids the somewhat tedious process of manually running the examples to make sure they work with every release.

Here's an example of what this looks like:
https://github.com/dapr/python-sdk/blame/master/examples/invoke-binding/README.md#L41-L47

Would you be willing to add these steps for at least the standalone (non-k8s) portion of your examples? You don't have to validate all of the output of your example, just enough to say "this example is working".

Here's the detailed guide for our example automation tool:
https://github.com/dapr/mechanical-markdown/tree/main/examples

Also, if you add your new example here, it will get run on every PR:
https://github.com/dapr/python-sdk/blob/master/tox.ini#L36-L45

To run validation locally:
```
pip3 install mechanical-markdown

cd examples/grpc_proxying
mm.py -l README.md
```

Reply:
let me give it a shot

Reply:
@wcs1only should be good now